lib/auth/webauthncli/fido2_common.go

```diff
@@ -71,9 +71,6 @@ type FIDO2DiagResult struct {
 // User interaction is required.
 func FIDO2Diag(ctx context.Context, promptOut io.Writer) (*FIDO2DiagResult, error) {
 	res := &FIDO2DiagResult{}
-	if !isLibfido2Enabled() {
```

Comment:
Hmm, this was actually dependent on the build tag. I think we'll have to keep the function, but it can be just a plain "return true" / "return false" depending on the build tags.

Sorry for the misleading suggestion, I had not spotted this line.